docs/deployment/plugins/k8s/index.rst

```diff
@@ -6,276 +6,8 @@ Configure Kubernetes Plugins
-                          value: "2"
-                      - projectQuotaMemory:
-                          value: "3000Mi"
-                  - development:
-                      - projectQuotaCpu:
-                          value: "4"
-                      - projectQuotaMemory:
-                          value: "5000Mi"
-                refresh: 5m
-              
-            Also add the following cluster resource templates to the ``~/.flyte/sandbox/cluster-resource-templates`` directory:
-
-            1. ``serviceaccount.yaml``
-
-            .. code-block:: yaml
-
-              apiVersion: v1
-              kind: ServiceAccount
-              metadata:
-                name: default
-                namespace: "{{ namespace }}"
-                annotations:
-                  eks.amazonaws.com/role-arn: "{{ defaultIamRole }}"
-            
-            2. ``spark_role.yaml``
-
-            .. code-block:: yaml
-
-              apiVersion: rbac.authorization.k8s.io/v1
-              kind: Role
-              metadata:
-                name: spark-role
-                namespace: "{{ namespace }}"
-              rules:
-                - apiGroups:
-                    - ""
-                  resources:
-                    - pods
-                    - services
-                    - configmaps
-                  verbs:
-                    - "*"
-
-            3. ``spark_service_account.yaml``
-
-            .. code-block:: yaml
-
-              apiVersion: v1
-              kind: ServiceAccount
-              metadata:
-                name: spark
-                namespace: "{{ namespace }}"
-                annotations:
-                  eks.amazonaws.com/role-arn: "{{ defaultIamRole }}"
-            
-            4. ``spark_role_binding.yaml``
-
-            .. code-block:: yaml
-
-              apiVersion: rbac.authorization.k8s.io/v1
-              kind: RoleBinding
-              metadata:
-                name: spark-role-binding
-                namespace: "{{ namespace }}"
-              roleRef:
-                apiGroup: rbac.authorization.k8s.io
-                kind: Role
-                name: spark-role
-              subjects:
-                - kind: ServiceAccount
-                  name: spark
-                  namespace: "{{ namespace }}"
-
-          .. group-tab:: Dask
-
-            Enable the Dask plugin on the demo cluster by adding the following block to ``~/.flyte/sandbox/config.yaml``:
-
-            .. code-block:: yaml
-
-              tasks:
-                task-plugins:
-                  default-for-task-types:
-                    container: container
-                    container_array: k8s-array
-                    sidecar: sidecar
-                    dask: dask
-                  enabled-plugins:
-                  - container
-                  - k8s-array
-                  - sidecar
-                  - dask
-
-        Start the demo cluster by running the following command:
```

Comment:
have you added this to the updated guide? after adding the config in the demo cluster, one needs to spin it up.

Reply:
I'll close off this PR to not interfere with that Jason Lai is doing and will add only the configs missing for Flyte core and binary (original purpose). I haven't been able to repro the Spark config on the flytectl demo cluster. Adding Helm overrides to something not deployed by Helm won't work.